modules/api-compatibility-exceptions.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * rest_api/understanding-compatibility-guidelines.adoc
+
+[id="api-compatibility-exceptions_{context}"]
+= API compatibility exceptions
+
+The following are exceptions to compatibility in {product-title}:
+
+[discrete]
+[id="OS-file-system-modifications-not-made_{context}"]
+== Operating system file system modifications not made with the Machine Config Operator
```

Comment:
As more discussion has evolved in this space, I think we should clarify:

"RHEL CoreOS file system modifications not made via a supported operator"